vscode-lean4/src/utils/leanInstaller.ts

```diff
@@ -161,6 +161,7 @@ export class LeanInstaller implements Disposable {
         }
 
         this.promptingInstallEmitter.fire(uri);
+        this.outputChannel.show(true);
```

Comment:
Are we certain this is the right change? It does not seem to introduce the 'Go to output' button.

Reply:
Oh, I missed the update at the bottom of the PR description.

Reply:
I'm also confused now, what happened to the show output button idea?  and show does this new outputChannel.show call relate to the new user configurable `lean4.autofocusOutput` option?  https://github.com/leanprover/vscode-lean4/pull/229

Reply:
Start failure messages should always be of interest to the user, so I don't think #229 applies here. But an explicit button isn't that bad either.